### DISCUSSION TITLE
"$pressed-bgColor" functionality

### DISCUSSION THREAD

**Discussion by PabloGLeiteDeSouza** [gluestack/gluestack-ui#1734] [Feature Request]
Hello people from the gluestack-ui community, I have some questions/possible bugs related to the functionality of the buttons and various components, "$pressed" it is not changing the values as it should change as soon as the component is pressed, I don't know if it has to be configured or If it really is a bug, I ask for help from more experienced people so that I can better understand what to do to resolve it.

```
<Button $pressed-bgColor="$green500" >
      <FontAwesome6 color="white" name="google" />
      <ButtonText ml="$2" >Login com o Google</ButtonText>
</Button>
```
  
![image](https://github.com/gluestack/gluestack-ui/assets/143195175/c37cf080-d6ef-437b-954f-0ab3226a4afc)

**Comment by Viraj-10** [comment]
Hey @PabloGLeiteDeSouza, you can try following code. 

```jsx
  <Button $active-bgColor="red" >
         <ButtonText ml="$2" >Login com o Google</ButtonText>
   </Button>
```

**Reply by PabloGLeiteDeSouza** [reply]
thank you :)
